src/Tool/DataObjectLoader.php

```diff
@@ -60,7 +60,7 @@ private function getAttributeNameFromParts(array $objectBrickParts,
 
     public function loadByAttribute(string $className,
                                     string $attributeName,
-                                    string $identifier,
+                                    ?string $identifier,
```

Comment:
alternatively, rather than widening the type here, I could modify the function to map `= ''` to `IS NULL`. Thoughts?

Reply:
@cancan101 That's a good idea. Could you please do that? TIA

Reply:
New PR: #458